### ISSUE TITLE
Delete a single facility from a list

### ISSUE THREAD

**Issue by dboyer** [open-apparel-registry/open-apparel-registry#478]
## Overview

List owners may delete a single facility from a list. 

### Is your feature request related to a problem? Please describe.

Currently, removing a facility from a list requires uploading an entire new list. It would be more efficient if users could remove a single facility from their list within the OAR admin interface.

### Describe the solution you'd like

When viewing the list, users have the option to delete a single line. They should receive a confirmation prompt that they would like to delete the facility.

**Comment by jwalgran** [comment]
We need to clarify whether we want to allow truly deleting an item from a list or if we want behavior that mimics as uploading a second full list that excludes the item. 

- Do we leave the facility on the map but do not display it as being affiliated with the facility?
- Do we need separate actions for true delete and making a single row inactive?

CC @nataliegrillon and @katieashaw

**Comment by katieashaw** [comment]
I stumbled across a few examples of why deleting single facilities would be useful yesterday. I was having a look at what facilities we have located in the UK, and there are several non-apparel facilities in there - see example in screenshot.

This is happening particularly with brands which have homeware departments, e.g. Next, Debenhams etc.

![Screenshot 2019-04-17 at 11 58 14](https://user-images.githubusercontent.com/46344729/56282936-44001700-6108-11e9-99f7-609074d3b3e9.png)
